spec/requests/as_account/playlist_spec.rb

```diff
@@ -121,9 +121,9 @@
 
       specify 'updates them replacing angle brackets with similar unicode characters accepted by YouTube' do
         expect(update).to be true
-        expect(playlist.title).to eq 'Yt Test ‹ ›'
-        expect(playlist.description).to eq '‹ ›'
-        expect(playlist.tags).to eq ['‹tag›']
+        expect(@my_playlist.title).to eq 'Yt Test ‹ ›'
+        expect(@my_playlist.description).to eq '‹ ›'
+        expect(@my_playlist.tags).to eq ['‹tag›']
```

Comment:
Is this change necessary? Existing playlist (`playlist` with `id`) should be enough to fetch info I think

Reply:
It wasn't working with `playlist`, and all the other tests in this section use `@my_playlist`. I didn't investigate much further than that.

Reply:
but it is testing if the playlist is updated or not, right? If it wasn't working then... I'm not sure, when I tried before this test was working

Reply:
Sure, but `update` runs on `@my_playlist`, not `playlist`. They're two different objects, so the updates won't be reflected on `playlist`.